### ISSUE TITLE
Add OSGI support to Manifest

### ISSUE THREAD

**Issue by zspitzer** [srikanth-lingala/zip4j#89]
zip4j is being used in the Lucee project, but Lucee requires an OSGI bundle, so currently it's being repackaged with an updated manifest file.  
https://github.com/lucee/extension-compress/tree/master/source/java/libs

Can OSGI support be added to the zip4j manifest? This what the Lucee version adds

```
Manifest-Version: 1.0
Bnd-LastModified: 1568382495891
Build-Jdk: 1.8.0_161
Built-By: mic
Bundle-Description: OSGI version of jar Zip4j
Bundle-License: http://www.apache.org/licenses/LICENSE-2.0.txt
Bundle-ManifestVersion: 2
Bundle-Name: org.lucee.zip4j
Bundle-SymbolicName: org.lucee.zip4j
Bundle-Version: 2.1.3.0
Created-By: Apache Maven Bundle Plugin
Export-Package: net.lingala.zip4j;uses:="net.lingala.zip4j.exception,net
 .lingala.zip4j.io.inputstream,net.lingala.zip4j.model,net.lingala.zip4j
 .progress";version="2.1.3",net.lingala.zip4j.crypto;uses:="net.lingala.
 zip4j.exception,net.lingala.zip4j.model,net.lingala.zip4j.model.enums";
 version="2.1.3",net.lingala.zip4j.crypto.PBKDF2;version="2.1.3",net.lin
 gala.zip4j.crypto.engine;uses:="net.lingala.zip4j.exception";version="2
 .1.3",net.lingala.zip4j.exception;version="2.1.3",net.lingala.zip4j.hea
 ders;uses:="net.lingala.zip4j.exception,net.lingala.zip4j.io.outputstre
 am,net.lingala.zip4j.model";version="2.1.3",net.lingala.zip4j.io.inputs
 tream;uses:="net.lingala.zip4j.model";version="2.1.3",net.lingala.zip4j
 .io.outputstream;uses:="net.lingala.zip4j.exception,net.lingala.zip4j.m
 odel";version="2.1.3",net.lingala.zip4j.model;uses:="net.lingala.zip4j.
 headers,net.lingala.zip4j.model.enums";version="2.1.3",net.lingala.zip4
 j.model.enums;version="2.1.3",net.lingala.zip4j.progress;version="2.1.3
 ",net.lingala.zip4j.tasks;uses:="net.lingala.zip4j.exception,net.lingal
 a.zip4j.headers,net.lingala.zip4j.io.inputstream,net.lingala.zip4j.mode
 l,net.lingala.zip4j.progress";version="2.1.3",net.lingala.zip4j.util;us
 es:="net.lingala.zip4j.exception,net.lingala.zip4j.io.inputstream,net.l
 ingala.zip4j.model,net.lingala.zip4j.model.enums,net.lingala.zip4j.prog
 ress";version="2.1.3"
Import-Package: javax.crypto,javax.crypto.spec,net.lingala.zip4j.crypto,
 net.lingala.zip4j.crypto.PBKDF2,net.lingala.zip4j.crypto.engine,net.lin
 gala.zip4j.exception,net.lingala.zip4j.headers,net.lingala.zip4j.io.inp
 utstream,net.lingala.zip4j.io.outputstream,net.lingala.zip4j.model,net.
 lingala.zip4j.model.enums,net.lingala.zip4j.progress,net.lingala.zip4j.
 tasks,net.lingala.zip4j.util
Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
Tool: Bnd-3.0.0.201509101326
```

**Comment by srikanth-lingala** [comment]
I have added `maven-bundle-plugin` to generate OSGI information in manifest. The manifest looks like this:

```
Manifest-Version: 1.0
Bnd-LastModified: 1571341427082
Build-Jdk: 1.8.0_192
Built-By: slingala
Bundle-Description: Zip4j - A Java library for zip files and streams
Bundle-License: https://www.apache.org/licenses/LICENSE-2.0.txt
Bundle-ManifestVersion: 2
Bundle-Name: Zip4j
Bundle-SymbolicName: net.lingala.zip4j
Bundle-Version: 2.2.3.5-SNAPSHOT
Created-By: Apache Maven Bundle Plugin
Export-Package: net.lingala.zip4j;uses:="net.lingala.zip4j.exception,n
 et.lingala.zip4j.io.inputstream,net.lingala.zip4j.model,net.lingala.z
 ip4j.progress";version="2.2.3",net.lingala.zip4j.crypto;uses:="net.li
 ngala.zip4j.exception,net.lingala.zip4j.model,net.lingala.zip4j.model
 .enums";version="2.2.3",net.lingala.zip4j.crypto.PBKDF2;version="2.2.
 3",net.lingala.zip4j.crypto.engine;uses:="net.lingala.zip4j.exception
 ";version="2.2.3",net.lingala.zip4j.exception;version="2.2.3",net.lin
 gala.zip4j.headers;uses:="net.lingala.zip4j.exception,net.lingala.zip
 4j.io.outputstream,net.lingala.zip4j.model";version="2.2.3",net.linga
 la.zip4j.io.inputstream;uses:="net.lingala.zip4j.model";version="2.2.
 3",net.lingala.zip4j.io.outputstream;uses:="net.lingala.zip4j.excepti
 on,net.lingala.zip4j.model";version="2.2.3",net.lingala.zip4j.model;u
 ses:="net.lingala.zip4j.headers,net.lingala.zip4j.model.enums";versio
 n="2.2.3",net.lingala.zip4j.model.enums;uses:="net.lingala.zip4j.exce
 ption";version="2.2.3",net.lingala.zip4j.progress;version="2.2.3",net
 .lingala.zip4j.tasks;uses:="net.lingala.zip4j.exception,net.lingala.z
 ip4j.headers,net.lingala.zip4j.io.inputstream,net.lingala.zip4j.model
 ,net.lingala.zip4j.progress";version="2.2.3",net.lingala.zip4j.util;u
 ses:="net.lingala.zip4j.exception,net.lingala.zip4j.io.inputstream,ne
 t.lingala.zip4j.model,net.lingala.zip4j.model.enums,net.lingala.zip4j
 .progress";version="2.2.3"
Import-Package: javax.crypto,javax.crypto.spec,net.lingala.zip4j.crypt
 o,net.lingala.zip4j.crypto.PBKDF2,net.lingala.zip4j.crypto.engine,net
 .lingala.zip4j.exception,net.lingala.zip4j.headers,net.lingala.zip4j.
 io.inputstream,net.lingala.zip4j.io.outputstream,net.lingala.zip4j.mo
 del,net.lingala.zip4j.model.enums,net.lingala.zip4j.progress,net.ling
 ala.zip4j.tasks,net.lingala.zip4j.util
Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
Tool: Bnd-4.2.0.201903051501
```

I will include this in the next release

**Comment by zspitzer** [comment]
thank you!

**Comment by srikanth-lingala** [comment]
Fixed in v2.2.3
